Tasks/Common/webdeployment-common/webconfigutil.ts

```diff
@@ -40,6 +40,9 @@ function addMissingParametersValue(appType: string, webConfigParameters) {
             'PYTHON_PATH': 'D:\\home\\python353x86\\python.exe',
             'PYTHON_WFASTCGI_PATH': 'D:\\home\\python353x86\\wfastcgi.py',
             'STATIC_FOLDER_PATH': 'static'
+        },
+        'Go': {
+            'GoExeFileName': ''
```

Comment:
just a suggestion, can we have some default value here / update during runtime?
For Django app, we made it empty, so that we can search for the settings file in runtime and update the value.
For Flask, bottle and node app, we have some default value (that may increase the success of the deployment)